app/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/LiabilityDetails.scala

```diff
@@ -42,7 +42,7 @@ case class LiabilityDetails(
   def isCompleted: Boolean =
     startDate.nonEmpty && expectedWeightNext12m.isDefined && newLiabilityStarted.isDefined && newLiabilityFinished.isDefined
 
-  def isInProgress: Boolean = exceededThresholdWeight.isDefined || expectToExceedThresholdWeight.isDefined
+  def isInProgress: Boolean = (exceededThresholdWeight.isDefined || expectToExceedThresholdWeight.isDefined || expectedWeightNext12m.isDefined) && !isCompleted
```

Comment:
Do we need a test to cover this scenario?

Reply:
Yes, just checked and the current tests don't cover then we only keep the **expectedWeightNext12m**

Reply:
Test added to LiabilityDetailsSpec